portugues/titanic/pular/boiar/computador.md~

```diff
@@ -0,0 +1,3 @@
+As pessoas não entedem muito bem como um computador chegou a costa em cima de um pedaco de madeira ainda funcionando. Como esse é o computador empresarial que tinha sido cedido a voce pela empresa ele é devolvido ao seu chefe. Ao ver os arquivos, ele nota que voce realmente trabalhou no que ele pediu. Assim que ele roda o seu código ele vê que está cheio de práticas ruins e não funciona muito bem. Você é demitido pós-mortem por causa de um aplicativo mal feito.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.